src/scanner.rs

```
@@ -2447,10 +2458,14 @@ impl<T: Iterator<Item = char>> Scanner<T> {
             return;
         }
         while self.indent > col {
-            let indent = self.indents.pop().unwrap();
```

Removing this unwrap would silence a potential error in the scanner logic.